library/junos_commit

```diff
@@ -142,6 +150,10 @@ def main():
         # --- UNREACHABLE ---
 
     try:
+        timeout = int(args['timeout'])
+        if timeout > 0:
+            dev.timeout = timeout
+
```

Comment:
@stacywsmith Can we change the logic where we can simply have 30 sec default value @ timeout=dict(required=False, default=0),
hence the code would change to
        timeout = int(args['timeout']) 
        dev.timeout = timeout

Say in this case i can also pass 10 sec as timeout (I know ppl wont opt for lower value). With you code snippet even if I will pass 10 it will use 30 sec as timeout.